install/package.management.js

```diff
@@ -364,10 +445,13 @@ module.exports = function (gulpWrapper, ctx) {
 		}
 
 		// Add tasks
-		taskArray.push('__npmInstall');
-		// if (ctx.type === "webApp") {
-			taskArray.push('__dedupeLibs');
-		// }
+		if (pluginYargs.yarn) {
+			taskArray.push('__yarnInstall');

```

Comment:
Are we assuming that Yarn is installed manually?

Reply:
yes, it was assumed that yarn was already installed, I will add a validation that checks if yarn is present and if it is not present maybe throw an exeception?